Yank/tests/test_repex.py

```diff
@@ -111,7 +111,7 @@ def test_replica_exchange(mpicomm=None, verbose=True):
 
     """
 
-    if verbose and ((not mpicomm) or (mpicomm.rank==0)): print "Testing replica exchange facility with harmonic oscillators: ",
+    if verbose and ((not mpicomm) or (mpicomm.rank==0)): print("Testing replica exchange facility with harmonic oscillators: ", end = " ")
```

Comment:
No, the syntax to supress the new line character changes between 2.X and 3.X, so we way not get it on this one line in either. I'll fix this on the next update.